controlpad/controller/required.js

```diff
@@ -95,45 +95,13 @@ function togglePopup() {
     overlay.style.display = isHidden ? 'block' : 'none';
 }
 
-// ------------------------- Orientation Check ----------------------------------
```

Comment:
These changes are fine. This file will also be a symlink pointing to a local required.js on the machine, but I was going to remove this code from here anyways so it's okay to be deleted